src/main/groovy/org/kordamp/gradle/plugin/jandex/JandexExtension.groovy

```diff
@@ -28,4 +29,13 @@ interface JandexExtension {
     Property<String> getVersion()
 
     Property<Integer> getIndexVersion()
+
+    /**
+     * Additional task names or patterns that should depend on the jandex task.
+     * The plugin already sets up dependencies for 'compileTestJava', 'jar', and Javadoc tasks.
+     * This property allows adding more tasks that should depend on the jandex task.
+     * You can specify exact task names or patterns using Ant-style patterns (e.g., '*Test').
+     * @return a list of task names or patterns
+     */
+    ListProperty<String> getAdditionalDependentTasks()
```

Comment:
I still disagree on adding this capability to the Jandex extension. It seems artificial.
Is this a recommended pattern? Otherwise every single custom task out there should provide the same capability.